File: backend/test/java/de/griefed/serverpackcreator/WebServiceTest.java
package de.griefed.serverpackcreator;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest(classes = WebServiceTest.class)
public class WebServiceTest {

  String[] args = new String[]{"--setup", "backend/test/resources/serverpackcreator.properties"};

  WebServiceTest() {
    ServerPackCreator.getInstance(
                         args)
                     .checkDatabase();
  }

  @Test
  void contextLoads() {
  }
}

File: serverpackcreator-app/src/test/kotlin/de/griefed/serverpackcreator/web/WebServiceTest.kt
package de.griefed.serverpackcreator.web

import de.griefed.serverpackcreator.api.ApiWrapper
import de.griefed.serverpackcreator.app.web.WebService
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import java.io.File

@SpringBootTest(classes = [WebServiceTest::class])
class WebServiceTest internal constructor() {

    init {
        println(File(".").absolutePath)
        WebService(
            ApiWrapper.api(
                File(
                    File("").absoluteFile.parent,
                    "serverpackcreator-api/src/test/resources/serverpackcreator.properties"
                )
            )
        )
    }


    @Test
    fun contextLoads() {

    }
}